src/components/Table/TableHead/TableHead.jsx

```diff
@@ -148,9 +188,68 @@ const TableHead = ({
   closeMenuText,
   lightweight,
   i18n,
+  hasResize,
 }) => {
   const filterBarActive = activeBar === 'filter';
+  const [columnWidth, setColumnWidth] = useState({});
+  const columnRef = ordering.map(() => createRef());
+  const columnVar = {
+    index: 0,
+    element: Node,
+    startX: 0,
+    move: 0,
+    validDrag: false,
+  };
+
+  const mousemoveCallback = e => {
+    const mousePosition = e.clientX + columnVar.startX;
+    const direction = e.clientX > columnVar.move ? 'right' : 'left';
+    const dragBounds = getColumnDragBounds(direction, columnWidth, columnVar.index);
+    columnVar.validDrag =
+      document.dir === 'rtl'
+        ? dragIsValidRtl(mousePosition, dragBounds)
+        : dragIsValidLtr(mousePosition, dragBounds);
+    columnVar.element.style.left = `${mousePosition}px`;
+    updateColumnResizeWrapper(columnVar.validDrag, columnVar.element);
```

Comment:
see above, I'd rather set a column state and have it declaratively choose the correct class accordingly

Reply:
Me too, but also same reply as above :-)

Reply:
done